src/message.rs

```diff
@@ -98,7 +97,7 @@ fn msg_send_super_fn<R: Any>() -> unsafe extern fn(*mut Object, Sel, ...) -> R {
     }
 }
 
-#[cfg(target_arch = "aarch64")]
+#[cfg(all(target_arch = "aarch64", not(feature = "gnustep_runtime")))]
 fn msg_send_fn<R>() -> unsafe extern fn(*mut Object, Sel, ...) -> R {
```

Comment:
`objc_msgSend` needs to be implemented as platform specific assembly. The GNUstep runtime has implementations for the following architectures:

* x86
* am64
* arm
* mips

We don't have an implementation for aarch64, so we don't want to define it here. Also, a mips implementation is still missing because I didn't have the time to figure out the calling conventions (specifically when to use `objc_msgSend_stret()`)